### ISSUE TITLE
cli error executing ember h g

### ISSUE THREAD

**Issue by marcemira** [dgeb/ember-engines#107]
Hey guys, I just tried to look at the cli's generate help after installing ember-engines and I got this:
```
$ ember h g
Requested ember-cli commands:

ember generate <blueprint> <options...>
  Generates new code from blueprints.
  aliases: g
  --dry-run (Boolean) (Default: false)
    aliases: -d
  --verbose (Boolean) (Default: false)
    aliases: -v
  --pod (Boolean) (Default: false)
    aliases: -p
  --classic (Boolean) (Default: false)
    aliases: -c
  --dummy (Boolean) (Default: false)
    aliases: -dum, -id
  --in-repo-addon (String) (Default: null)
    aliases: --in-repo <value>, -ir <value>

Cannot find module 'ember-cli/blueprints/route/index'
Error: Cannot find module 'ember-cli/blueprints/route/index'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/path/to/my/project/node_modules/ember-engines/blueprints/in-repo-engine/index.js:5:13)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
```
this is my package.json:
```
{
  "name": "my-app",
  "version": "0.0.0",
  "description": "Small description for my-app goes here",
  "private": true,
  "directories": {
    "doc": "doc",
    "test": "tests"
  },
  "scripts": {
    "build": "ember build",
    "start": "ember server",
    "test": "ember test"
  },
  "repository": "",
  "engines": {
    "node": ">= 0.10.0"
  },
  "author": "",
  "license": "MIT",
  "devDependencies": {
    "broccoli-asset-rev": "^2.4.2",
    "ember-ajax": "^2.0.1",
    "ember-cli": "2.6.0-beta.2",
    "ember-cli-app-version": "^1.0.0",
    "ember-cli-babel": "^5.1.6",
    "ember-cli-dependency-checker": "^1.2.0",
    "ember-cli-fastboot": "^1.0.0-beta.1",
    "ember-cli-htmlbars": "^1.0.3",
    "ember-cli-htmlbars-inline-precompile": "^0.3.1",
    "ember-cli-inject-live-reload": "^1.4.0",
    "ember-cli-jshint": "^1.0.0",
    "ember-cli-postcss-fixed": "0.0.6",
    "ember-cli-qunit": "^1.4.0",
    "ember-cli-release": "0.2.8",
    "ember-cli-sri": "^2.1.0",
    "ember-cli-uglify": "^1.2.0",
    "ember-data": "^2.6.0-beta.1",
    "ember-engines": "0.2.2",
    "ember-export-application-global": "^1.0.5",
    "ember-load-initializers": "^0.5.1",
    "ember-network": "0.3.0",
    "ember-resolver": "^2.0.3",
    "ember-responsive": "1.2.6",
    "ember-truth-helpers": "1.2.0",
    "liquid-fire": "0.23.1",
    "loader.js": "^4.0.1"
  },
  "ember-addon": {
    "paths": [
      "lib/engine-main",
      "lib/engine-dashboard"
    ]
  },
  "dependencies": {
    "autoprefixer": "^6.3.6",
    "postcss-cssnext": "^2.5.2",
    "postcss-import": "^8.1.2"
  }
}
```

**Comment by rwjblue** [comment]
The route blueprint is no longer present from ember-cli itself (it has been extracted into another package in preparation of moving them into Ember themselves).

**Comment by trentmwillis** [comment]
This should be fixed now via #147.

**Comment by marcemira** [comment]
awesome :+1:
